### ISSUE TITLE
Fetch more than 30 forks

### ISSUE THREAD

**Issue by Newgap** [techgaun/active-forks#13]
Forks from GitHub are paged 30 at a time. It would be nice to fetch them all for large projects (many forks).

**Comment by techgaun** [comment]
@Newgap thats a good point. Would be open for a PR from community as I have couple of busy weeks.

**Comment by Syed-Umair** [comment]
I will work on this.

**Comment by techgaun** [comment]
all yours @Syed-Umair

**Comment by Syed-Umair** [comment]
@techgaun, Do you need me to load specific amount of forks data like 100 or all that is available.

**Comment by techgaun** [comment]
@Syed-Umair I think the best would be to follow the guidelines and put lazy loading or page numbering at the bottom of the page: `https://developer.github.com/v3/guides/traversing-with-pagination/`

**Comment by Syed-Umair** [comment]
Ok, I will do that.

**Comment by Newgap** [comment]
Don't forget that if we change the sorting, you will need to refetch the data from GitHub with that new sorting. Or, you can fetch all data in memory and sort that instead.

**Comment by techgaun** [comment]
@Newgap agreed. I am however not sure if we really want to fetch all data. For examle, `facebook/react` has 20K forks and not sure how helpful/useful would it be to pull them all. But, again, for something like such repos, this tool might not be used as often and could be potential trade-off? Just not sure what would be the best way to get around this. Thoughts?

**Comment by ModischFabrications** [comment]
Can we set the number of forks to be fetched via UI instead of using a max of 100 like in #14? I usually use this tool to find the fork with the latest commit, which is impossible or at least very luck-based with repos with more than 100 forks.

**Comment by techgaun** [comment]
@ModischFabrications fair point. That's probably a good idea. Will try to get that in soon

**Comment by techgaun** [comment]
Fixed by #99: forks are now fetched 100 per page following the `Link` response headers — up to 400 forks without a token and 3,000 with a personal access token (field below the search box) — with a notice when results are truncated.
